src/main/resources/templates/symbol.html

```diff
@@ -9,7 +8,7 @@
     </head>
     <body>
         <h1>Market Data</h1>
-        <h2 id="symbol"></h2>
+        <h2 id="symbol" aria-label="Interactive Ticker"></h2>
```

Comment:
I added an aria-label to this h2 because validation was complaining that it was an empty element. I understand that this is being used for innerHTML, so I didn't want to put anything inside of the h2. 

The reason why the validator is complaining is that empty headers pose issues for screen readers. By adding an aria-label, this solves the issue of screen readers not being able to figure out the empty element.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label